### ISSUE TITLE
Membership may be assigned prior to capturing payment

### ISSUE THREAD

**Issue by hismightiness** [HotcakesCommerce/hotcakes-commerce-core#213]
## Describe the bug
Discovered and based on Issue #189 

## Software Versions
- DNN:  (see linked issue)
- Hotcakes:  (see linked issue)

## To Reproduce
1. (see linked issue)
2. Only, set the payment gateway to authorize instead of charging the full amount at checkout.

## Expected behavior
The order is placed and payment is pending since it's only authorized.  The membership is not yet assigned.  

## Actual behavior
The order is placed and payment is pending, but the membership is prematurely assigned.  \

**Comment by parasdaryanani** [comment]
Is payment capture readily detected within HCC by default?

**Comment by hismightiness** [comment]
I don't think this is an issue any longer.  I don't remember why I created this issue and linked it to the original.
